javalib/src/main/scala/java/util/function/BinaryOperator.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package java.util.function
+
+import scala.scalajs.js.annotation.JavaDefaultMethod
+
+import java.util.{Comparator, Objects}
+
+trait BinaryOperator[T] extends BiFunction[T, T, T]
+
+object BinaryOperator {
+  @JavaDefaultMethod
+  def minBy[T](comparator: Comparator[_ >: T]): BinaryOperator[T] = (a: T, b: T) => {
```

Comment:
I think the braces need to go right after the `=` sign (except if you make it a single liner)